### PR TITLE
Publishing workflow

### PR DESCRIPTION
**Pull request by Edoardo-Pedicillo**
In this PR, @MatteoRobbiati @AleCandido and I have deployed the workflow to publish the`latest`and `stable` Qibo's documentation, and we have made minor fixes to the `conf.py` file.
Checklist:
- [ ] Reviewers confirm new code works as expected.
- [ ] Tests are passing.
- [ ] Coverage does not decrease.
- [ ] Documentation is updated.

### CODE REVIEW

**Comment by scarrazza** [comment]
@Edoardo-Pedicillo could you please have a look at pre-commit?

**Review by andrea-pasquale — COMMENTED** [review]
Thanks @Edoardo-Pedicillo.
I would suggest to squash some commits. 124 seems quite a lot to me.

**Comment by AleCandido** [comment]
> Thanks @Edoardo-Pedicillo.
> I would suggest to squash some commits. 124 seems quite a lot to me.

They are mostly "trigger" commits, so almost empty (something like one newline added or removed), they should be easy to squash.

If you know how to do @andrea-pasquale, can you try doing it yourself. Otherwise @Edoardo-Pedicillo you can try, and if you have troubles or is taking too long I will do.

**Comment by Edoardo-Pedicillo** [comment]
I have squashed the commits, and only a few of them remain because the rebase generates conflicts.

**Comment by Edoardo-Pedicillo** [comment]
Now the pre-commit checks have passed, thanks @andrea-pasquale

File: doc/source/conf.py [hunk 1]
```diff
@@ -16,16 +16,15 @@
 from recommonmark.transform import AutoStructify
 
 sys.path.insert(0, os.path.abspath(".."))
-import qibo
 
 # -- Project information -----------------------------------------------------
 
-project = "qibo"
+project = " "
```
Comment (scarrazza) [inline doc/source/conf.py:22]: Please follow the approach in https://github.com/qiboteam/qibo/pull/745, or simply remove those changes in this PR.

Comment (MatteoRobbiati) [reply doc/source/conf.py:22]: I suggest to remove them, since I am making these changes in dedicated PR.

**Comment by scarrazza** [comment]
@MatteoRobbiati thanks but could you please reset conf.py to master so we avoid extra changes?

**Comment by MatteoRobbiati** [comment]
> @MatteoRobbiati thanks but could you please reset conf.py to master so we avoid extra changes?

Now the `conf.py` file in our fork is identical to `qibo`'s master one. I think this can fix the differences.

**Pull request merged**